packages/cli/src/dev.ts

```diff
@@ -31,7 +31,7 @@ function createSchema(generator: TJS.SchemaGenerator, symbol: string) {
     return { components: { schemas: {} } }
   }
 
-  let fixedSymbol = symbol.replace('Array<', '').replace('>', '')
+  let fixedSymbol = symbol.replace(/Array\<(.*)\>/gm, '$1')
```

Comment:
@mirceanis Just wanted to highlight this change outside of the kv-store package. When testing a bit with the schema-generator I noticed that it mangled anything with carets in there, whilst it was only intended for Arrays.